### DISCUSSION TITLE
PORN

### DISCUSSION THREAD

**Discussion by michalsieron** [super-fajna-ogranizacja/Zastosowanie-Informatyki-w-Gospodarce#34] [Projects]
### Description

Welcome to PORN.
Feel free to browse the site's infrastructure and unearth knowledge to satisfy your curiosity.

### URLs

https://porn.jules-aubert.info/

### Types

- [ ] CLI
- [ ] TUI
- [ ] GUI
- [X] webapp
- [ ] library
- [ ] framework

### Platforms

- [ ] Linux
- [ ] Windows
- [ ] MacOS
- [ ] iOS
- [ ] Android
- [X] Web

### Categories

Using now and then

### 
---

Great content

**Comment by rodakrafal** [comment]
Ale super projekt

**Comment by rodakrafal** [comment]
I love PORN
